Tests/Resource/ResourceTests.swift

```diff
@@ -24,48 +24,35 @@ extension Todo: ContentMappable {
 extension Todo: StructuredDataRepresentable {
     public var structuredData: StructuredData {
         return [
-            "id": id.map({StructuredData.from($0)}) ?? nil,
-            "title": StructuredData.from(title),
-            "done": StructuredData.from(done)
+            "id": id.map({StructuredData($0)}) ?? nil,
+            "title": StructuredData(title),
+            "done": StructuredData(done)
```

Comment:
This is the stupidest thing to comment on, but just so you know this can be
```swift
return [
    "id": id.map { .from($0) } ?? nil,
    "title": .from(title),
    "done": .from(done)
]
```